src/liboslexec/batched_backendllvm.h

```diff
@@ -114,7 +114,7 @@ class BatchedBackendLLVM : public OSOProcessorBase {
     /// (no conversion is performed if cast is the default of UNKNOWN).
     llvm::Value* llvm_load_value(llvm::Value* ptr, const TypeSpec& type,
                                  int deriv, llvm::Value* arrayindex,
-                                 int component,
+                                 int component, bool ptr_is_uniform,
```

Comment:
`bool ptr_is_uniform`

I found the name to be confusing, because varying pointers exist and this is really saying what the ptr points to is uniform.

In the store I saw `dst_is_uniform` and it made total sense, so maybe for the loads we could just use
`bool src_in_uniform`
or something instead

Reply:
Ok, I will rename `ptr` to `src_ptr` and `ptr_is_uniform` to `src_is_uniform`.